django/tasks/__init__.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.connection import BaseConnectionHandler, ConnectionProxy
+from django.utils.module_loading import import_string
+
+from . import checks, signal_handlers  # noqa
```

Comment:
I wonder if this is not better done from an `AppConfig.ready` method? This way if the app is not installed, these modules are not imported, even if `django.tasks` is imported.